src/index.tsx

```diff
@@ -79,6 +79,10 @@ function useGetLatest<Value>(value: Value): () => Value {
   return useCallback(() => ref.current, []);
 }
 
+export type Reset = () => void;
+
+export type MutationResult<Input = any, Data = any, Error = any> = [(input: Input) => Promise<Data | undefined>, { status: Status, data?: Data, error?: Error, reset: Reset }];
```

Comment:
```suggestion
export type MutationResult<Input, Data, Error> = [(input: Input) => Promise<Data | undefined>, { status: Status, data?: Data, error?: Error, reset: Reset }];
```
This doesn't need to define a default for the generics, the useMutation will do it already